examples/inverse/plot_dics_beamformer.py

```diff
@@ -53,7 +53,8 @@
 evoked = epochs.average()
 
 # Read forward operator
-forward = mne.read_forward_solution(fname_fwd, surf_ori=True)
+forward = mne.read_forward_solution(fname_fwd)
+forward = mne.convert_forward_solution(forward, surf_ori=True)
```

Comment:
isn't the conversion to surf ori now done automatically in the inverse solvers if 0 < loose < 1 ?

maybe we can skip a few of these conversion lines?